<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans Light</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 208, 255);</string>
  </property>
  <widget class="QLineEdit" name="path_le">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_dir_btn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите директорию файла</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_le">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите название книги</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
